GotExplorer.API/Extensions/ServiceResultToActionResultExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using GotExplorer.BLL.Services.Results;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.WebUtilities;
+using System.Collections;
+using System;
+
+namespace GotExplorer.API.Extensions
+{
+    public static class ServiceResultToActionResultExtensions
```

Comment:
Do not use Problemdetails at all. Please provide details as a validation result from fluent api